doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -109,7 +109,7 @@ Amazon Sidewalk
 Bluetooth® LE
 -------------
 
-|no_changes_yet_note|
+* The :c:func:`hci_vs_sdc_llpm_mode_set` function returns an error if Low Latency Packet Mode (LLPM) is not supported or not enabled in the Bluetooth LE controller configuration (:kconfig:option:`CONFIG_BT_CTLR_SDC_LLPM`).
```

Comment:
I think we need to start changelog entries with the following verbs Added/Removed/Updated: 

```suggestion
* Updated the Bluetooth LE Controller to make the :c:func:`hci_vs_sdc_llpm_mode_set` function return an error if Low Latency Packet Mode (LLPM) is not supported or not enabled in the Bluetooth LE controller configuration (:kconfig:option:`CONFIG_BT_CTLR_SDC_LLPM`).
```

Reply:
I think we should refer to `SoftDevice Bluetooth LE Controller driver` after the rephrase